collections/src/main/scala/strawman/collection/IndexedView.scala

```diff
@@ -0,0 +1,84 @@
+package strawman
+package collection
+
+import scala.{Any, Boolean, Equals, IllegalArgumentException, Int, NoSuchElementException, Nothing, annotation, IndexOutOfBoundsException, throws, AnyRef, Array, deprecated, `inline`}
+import scala.Predef.{<:<, intWrapper}
+
+/** View defined in terms of indexing a range */
+trait IndexedView[+A] extends IndexedSeqOps[A, View, View[A]] with SeqView[A] { self =>
```

Comment:
Should we call it `IndexedSeqView`?

Reply:
I agree that it would make more sense.

Reply:
I am not sure that you can have a sensible indexing without a seq, so to me `IndexedView` (and, indeed, `IndexedOps`) make just as much sense to me.